### PR TITLE
CI Test

### PR DESCRIPTION
**Pull request by ricklove**
- Testing CI
- Fixed pre-commit command

### CODE REVIEW

File: .github/workflows/main.yml [hunk 1]
```diff
@@ -64,7 +64,7 @@ jobs:
         env:
           TAQ_VERSION: "{{ github.ref_name }}"
         run: |
-          deno compile --output taq --target $DENO_TARGET --allow-run --allow-write --allow-read --allow-env index.ts --quickstart "`cat quickstart.md`" --setBuild ${{ github.ref_name }} --setVersion ${{ github.ref_name }}
+          DENO_DIR=./deno deno compile --output taq --target $DENO_TARGET --allow-run --allow-write --allow-read --allow-env index.ts --quickstart "`cat quickstart.md`" --setBuild ${{ github.ref_name }} --setVersion ${{ github.ref_name }}
```
Comment (GImbrailo) [inline .github/workflows/main.yml:67]: I tested without the using the `DENO_DIR` variable. Is there a reason why we want to use this variable?

File: .github/workflows/main.yml [hunk 2]
```diff
@@ -54,7 +54,7 @@ jobs:
       - name: Setup Deno
         uses: denoland/setup-deno@v1
         with:
-          deno-version: "^1.0.0"
+          deno-version: "1.18.2"
```
Comment (GImbrailo) [inline .github/workflows/main.yml:57]: There is another section in this file where we need to change the version on line 151

File: .github/workflows/main.yml (hunk 1 above)
Comment (GImbrailo) [reply .github/workflows/main.yml:67]: @mweichert

File: .github/workflows/main.yml (hunk 2 above)
Comment (GImbrailo) [reply .github/workflows/main.yml:57]: @mweichert

File: .github/workflows/main.yml (hunk 1 above)
Comment (GImbrailo) [reply .github/workflows/main.yml:67]: Sounds good

File: .github/workflows/main.yml (hunk 2 above)
Comment (mweichert) [reply .github/workflows/main.yml:57]: Updated.

**Pull request merged**